src/mantine-core/src/components/Progress/Progress.test.tsx

```diff
@@ -23,7 +23,12 @@ describe('@mantine/core/Progress', () => {
   itSupportsSx(Progress, defaultProps);
   itSupportsRef(Progress, defaultProps, HTMLDivElement);
   checkAccessibility([mount(<Progress value={80} aria-label="test-progress" />)]);
-  itSupportsStylesApi(Progress, defaultProps, Object.keys(ProgressStylesApi), 'Progress');
+  itSupportsStylesApi(
```

Comment:
Please remove this test entirely

Reply:
Are you saying to remove the `itSupportsStylesApi` test entirely for `Progress` (why ?) or just the changes _I made to it_ to pass the tests ? (I am still coming to terms on understating the test suite, but I am assuming this has something to do with the generated Mantine classes on components).

The rest of the below mentioned changes will be taken care.

Reply:
`itSupportsStylesApi` was completely removed see https://github.com/mantinedev/mantine/commit/4b4943af00e6ecbfbb49030abed6354f2989314d

Reply:
Hmm.. Okay. Wasn't aware it was removed for all components.